### ISSUE TITLE
Error 500 at build time

### ISSUE THREAD

**Issue by rktmatt** [s00d/nuxt-i18n-micro#28]
At build time, the '/_locales/' folder get pre-rendered by nitro. But it fails and there is no explanation as to why.

Example on pnpm build : 
```
ℹ Prerendering 40 routes                                                                                                                                   nitro 9:10:13 PM
  ├─ /_locales/about/en/data.json (17ms)                                                                                                                    nitro 9:10:13 PM
  │ └── Error: [500]
  ├─ /_locales/contact/en/data.json (14ms)                                                                                                                  nitro 9:10:13 PM
  │ └── Error: [500]
  ├─ /_locales/dashboard/en/data.json (14ms)                                                                                                                nitro 9:10:13 PM
  │ └── Error: [500]
```
...etc and lower on the same command :
```
Errors prerendering:
  ├─ /_locales/about/en/data.json (17ms)                                                                                                                    nitro 9:10:13 PM
  │ └── Error: [500]
  ├─ /_locales/contact/en/data.json (14ms)                                                                                                                  nitro 9:10:13 PM
  │ └── Error: [500]
  ├─ /_locales/dashboard/en/data.json (14ms)
```
...etc

```
ERROR  Exiting due to prerender errors.                                                                                                                          9:10:13 PM

  at prerender (node_modules/.pnpm/nitropack@2.9.7_@opentelemetry+api@1.9.0_drizzle-orm@0.33.0_@cloudflare+workers-types@4.20240_bhl4qsy7253l5px3bgqyitvdqa/node_modules/nitropack/dist/chunks/prerender.mjs:220:11)
  at async node_modules/.pnpm/nuxt@3.13.2_@opentelemetry+api@1.9.0_@parcel+watcher@2.4.1_@types+node@20.14.9_drizzle-orm@0._5izbfushtltydskmchrenphyii/node_modules/nuxt/dist/index.mjs:3628:5
  at async build (node_modules/.pnpm/nuxt@3.13.2_@opentelemetry+api@1.9.0_@parcel+watcher@2.4.1_@types+node@20.14.9_drizzle-orm@0._5izbfushtltydskmchrenphyii/node_modules/nuxt/dist/index.mjs:5778:3)
  at async Object.run (node_modules/.pnpm/nuxi@3.13.2/node_modules/nuxi/dist/chunks/build.mjs:102:5)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.13.2/node_modules/nuxi/dist/shared/nuxi.b8b195e1.mjs:1648:16)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.13.2/node_modules/nuxi/dist/shared/nuxi.b8b195e1.mjs:1639:11)
  at async runMain$1 (node_modules/.pnpm/nuxi@3.13.2/node_modules/nuxi/dist/shared/nuxi.b8b195e1.mjs:1777:7)



 ERROR  Exiting due to prerender errors.                                                                                                                          9:10:13 PM

 ELIFECYCLE  Command failed with exit code 1.

```

**Comment by s00d** [comment]
Hello. I need a code example, it's not clear what the problem is.

**Comment by rktmatt** [comment]
Thank you for your quick response. Unfortunately I tried to make a reproduction, but was unable to do so because I have no idea where the bug comes from and the error messages didn't help.

I thought it might be because nuxt hub was building differently, but it is not the case on my reproduction attempts.

When trying builds on the project, I tried to remove all the /locales/ folder except where there was translations. Because not every route did Error. It didn't affect the issue.

Most of the translation jsons were empty as I had not optimized the translations into separate fields at this stage.

Unfortunately I migrated sunday evening to the normal i18n because I needed to publish this small project quickly.

Why would prerendering json fail is an interesting question though.
